### PR TITLE
Update non-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the document to add Plan 1 and 2 as Plan 1 supports macOS and android https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/defender-endpoint-plan-1?view=o365-worldwide#cross-platform-support

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7609

File: microsoft-365/security/defender-endpoint/non-windows.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ ms.technology: mde
 [!INCLUDE [Microsoft 365 Defender rebranding](../../includes/microsoft-defender.md)]
 
 **Applies to:**
-- [Microsoft Defender for Endpoint Plan 2](https://go.microsoft.com/fwlink/p/?linkid=2154037)
+- [Microsoft Defender for Office 365 plan 1 and plan 2](defender-for-office-365.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/non-windows.md]:
```suggestion
- [Microsoft Defender for Office 365 plan 1 and plan 2](/microsoft-365/security/office-365-security/defender-for-office-365)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder. Thanks

**Comment by mjcaparas** [comment]
I don't think it's office 365 - Microsoft Defender for Office 365 plan 1 and plan 2. Shouldn't it be Defender for Endpoint Plan 1 and Plan 2? @msbemba

**Comment by msbemba** [comment]
Thank you, @mjcaparas That is correct. I have made the changes to the PR and it should be good now.

Thank you
Sri

**Pull request merged**